pkg/clustermanager/eksa_installer.go

```diff
@@ -38,7 +38,7 @@ func NewEKSAInstaller(client KubernetesClient, reader manifests.FileReader) *EKS
 }
```

Comment:
Do we need to update the KubernetesClient as well? I believe that's a retrier client as well.

Reply:
oh yea good catch